packages/ethereum-contracts/test/ops-scripts/deployment.test.js

```diff
@@ -225,143 +225,132 @@ contract("Embedded deployment scripts", (accounts) => {
                 }
             });
 
-            // TODO deployment script upgrades detection only works for truffle
-            if (process.env.IS_TRUFFLE) {
```

Comment:
**Low**: Hardhat can run these tests too, I tried locally when attempting to fix the contract logic change issue.